### ISSUE TITLE
Differences between db-data.xml and db-data-lbts.xml

### ISSUE THREAD

**Issue by timler** [jembi/bsis#433]
There are some differences in the files db-data.xml and db-data-lbts.xml. We are using the latter in development and the former hasn't always been kept up to date with changes.

Some issues noted in db-data.xml:
- change sets that should be removed:
 - 20141117-1128 and 20141117-1133 should be removed (they are to do with donor codes)
- change sets that are missing:
 - 201509171503 - update PackType with testSampleProduced = 1 except for 'Did not bleed'
 - 20151005-1101 - insert data BloodTests: HIV, HBV, HCV, Syphilis confirmatory
 - 20151006-1200 - update data BloodTest: set flagComponentsForDiscard=1 where category=TTI & insert data BloodTest 'Antibody Screen'

**Comment by timler** [comment]
Maybe db-data.xml should just be deleted?

**Comment by danfuterman** [comment]
Yes, pretty certain it can be deleted. The lbts data has become the baseline dataset now, I initially kept them both because I wasn't sure how much the lbts dataset would differ from the original legacy data.

If we could rename db-data-lbts.xml to db-data.xml afterwards, that would be good too.
